src/models/contracts.js

```diff
@@ -16,7 +18,7 @@ const ContractsSchema = new Schema({
     type: ObjectId,
     required: true
   },
-  timeStamp: {
+  createdAt: {
```

Comment:
Just to clarify what the timestamp represents, and not to expose implementation.